plugins/module_utils/v4/lcm/helpers.py

```diff
@@ -0,0 +1,68 @@
+# Copyright: (c) 2024, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ..utils import raise_api_exception  # noqa: E402
+
+
+def get_lcm_status(module, api_instance, cluster_ext_id):
```

Comment:
```suggestion
def get_lcm_status(module, api_instance, cluster_ext_id=None):
```
Please do it for other helpers also if cluster external ID is not mandatory